src/installer/pkg/snap/dotnet-runtime-7.0/snap/snapcraft.yaml

```diff
@@ -0,0 +1,44 @@
+name: dotnet-runtime-70
+base: core18
+version: 7.0.0-preview.1.22076.8
+summary: Cross-Platform .NET Core Runtime. 
```

Comment:
Our branding is .NET Runtime. We have dropped Core.

Also “cross-platform” feels redundant in this description